NEWS.md

```diff
@@ -22,6 +20,10 @@ been removed from the docs but are still supported for backwards compatibility.
 - Pacman's `SigLevel` option was changed to use the same default value as used on Arch
 which is `SigLevel = Required DatabaseOptional`. If this results in keyring errors,
 you need to update the keyring by running `pacman-key --populate archlinux`.
+- Support for CentOS 7 was dropped.
+- Support for BIOS/grub was dropped. To allow users to configure grub themselves, the
+new `--bios-size` option can be used to add a BIOS boot partition of the specified size
+on which grub can be installed.
```

Comment:
```suggestion
on which grub (or any other bootloader) can be installed with the help of mkosi's script support.
```

Reply:
Maybe the unconditional copying out of the kernel and initrd should be mentioned as well, since for most users this will be new output.